docs/root/source/timestamps.md

```diff
@@ -36,35 +34,26 @@ In all likelihood, you will never have to worry about leap seconds because they
 There's another gotcha with (Unix time) timestamps: you can't calculate the real-world elapsed time between two timestamps (correctly) by subtracting the smaller timestamp from the larger one. The result won't include any of the leap seconds that occured between the two timestamps. You could look up how many leap seconds happened between the two timestamps and add that to the result. There are many library functions for working with timestamps; those are beyond the scope of this documentation.
 
 
-## Avoid Doing Transactions Around Leap Seconds
-
-Because of the ambiguity and confusion that arises with Unix time around leap seconds, we advise users to avoid creating transactions around leap seconds.
-
-
 ## Interpreting Sets of Timestamps
 
 You can look at many timestamps to get a statistical sense of when something happened. For example, a transaction in a decided-valid block has many associated timestamps:
 
-* its own timestamp
-* the timestamps of the other transactions in the block; there could be as many as 999 of those
 * the timestamp of the block
 * the timestamps of all the votes on the block
 
-Those timestamps come from many sources, so you can look at all of them to get some statistical sense of when the transaction "actually happened." The timestamp of the block should always be after the timestamp of the transaction, and the timestamp of the votes should always be after the timestamp of the block.
-
 
 ## How BigchainDB Uses Timestamps
 
 BigchainDB _doesn't_ use timestamps to determine the order of transactions or blocks. In particular, the order of blocks is determined by RethinkDB's changefeed on the bigchain table.
 
-BigchainDB does use timestamps for some things. It uses them to determine if a transaction has been waiting in the backlog for too long (i.e. because the node assigned to it hasn't handled it yet). It also uses timestamps to determine the status of timeout conditions (used by escrow).
+BigchainDB does use timestamps for some things. It uses them to determine if a transaction has been waiting in the backlog for too long (i.e. because the node assigned to it hasn't handled it yet).
```

Comment:
There's a special `assignment_timestamp` added to a transaction when it gets written to the backlog table. It's what's used to decide if a transaction is stale.

@libscott Maybe this paragraph should say something about that (i.e. to distinguish the two kinds of timestamps)? We're not getting rid of the `assignment_timestamp`.

Note: The `assignment_timestamp` is removed before the transaction gets written to a block. It's an ephemeral backlog-table-specific appendage.